docs/guides/jumpgates.md

```diff
@@ -0,0 +1,39 @@
+# How to Bridge Tokens through Jumpgate
+
+Jumpgates are a class of contracts that faciliate cross-chain token transfers under DAO operations. Each jumpgate is set up to work with a particular token and a pre-defined recipient. Below is the procedure of transferring tokens using a jumpgate.
+
+[**Watch video tutorial**](https://youtu.be/IqphF28aTUU)
+
+### 1. Verify Jumpgate
+In this step we will be making sure that the jumpgate is correctly configured. You will only need to do this once because jumpgates are non-upgradeable contracts.
+
+Go to [Etherscan](https://etherscan.io/)  and open the Jumpgate page. Click the "Contract" tab, the green check mark confirms that the source code is verified. Check the parameters:
+- `arbiterFee` is always 0;
+- `bridge` is the address of the bridge. If the Jumpgate is using Wormhole bridge, you can check the address against the [Wormhole docs](https://book.wormhole.com/reference/contracts.html); 
```

Comment:
Better specify which bridge (token bridge) to look for; can list the current address also